common/security_request_error.md

```diff
@@ -17,6 +17,10 @@ Understanding the `SecurityRequestError` exception.
 
 When the **Return Code** of a **Security Result** returned by IRRSMO00 is **NOT** equal to `0`, a `SecurityRequestError` will be raised to indicate that the request failed. A `SecurityRequestError` can be handled as follows.
 
+&nbsp;
+
+For an understanding of return or reason codes returned from IRRSMO00, you can review [IRRSMO00's Return and Reason Codes documentation]](https://www.ibm.com/docs/en/zos/2.3.0?topic=operations-return-reason-codes). The 'Messages' list shown in the sample below also contains any error messages or other such information. 

```

Comment:
```markdown
{: .note}
> _For an understanding of IRRSMO00 return and reason codes, see the [IRRSMO00 Return and Reason Codes](https://www.ibm.com/docs/en/zos/3.2.0?topic=operations-return-reason-codes) documentation._
```

Also move this "Note" annotation above the "Warning" annotation.

Reply:
section now formatted as:

```
&nbsp;

{: .warning }
> _A **Return Code** of anything other than `0` from IRRSMO00 is indicative of a failure with one or more of the operations performed by IRRSMO00, and pyracf will always raise a `SecurityRequestError` to bring attention to these failures._

&nbsp;

{: .note}
> _For an understanding of IRRSMO00 return and reason codes, see the [IRRSMO00 Return and Reason Codes](https://www.ibm.com/docs/en/zos/3.2.0?topic=operations-return-reason-codes) documentation._

&nbsp;

When the **Return Code** of a **Security Result** returned by IRRSMO00 is **NOT** equal to `0`, a `SecurityRequestError` will be raised to indicate that the request failed. A `SecurityRequestError` can be handled as follows.

&nbsp;
```

Reply:
Place the "Note" section above the "Warning" section.

Reply:
Change `pyracf` to `pyRACF` in "Warning" section.

Reply:
changed.